exercises/practice/zebra-puzzle/.meta/example.ex

```diff
@@ -0,0 +1,192 @@
+defmodule ZebraPuzzle do
+  @nationalities ~w(englishman norvegian ukrainian japanese spaniard)a
```

Comment:
A small typo (also in the test):
```suggestion
  @nationalities ~w(englishman norwegian ukrainian japanese spaniard)a
```